drivers/video/Kconfig

```diff
@@ -50,6 +50,10 @@ config VIDEO_BUFFER_SMH_ATTRIBUTE
 	  1: SMH_REG_ATTR_NON_CACHEABLE
 	  2: SMH_REG_ATTR_EXTERNAL
 
+config VIDEO_BUFFER_ALLOC_TIMEOUT
```

Comment:
I don't see where do you use this new config ? Also, this change should not be in the same commit with the change in esp32 power management.